SOURCES/etc/xapi.d/plugins/xcpngutils/__init__.py

```diff
@@ -63,12 +63,24 @@ def log_unhandled_exception(origin, exception_type, exception_value,
     return logger
 
 
+class ProcessException(Exception):
+    def __init__(self, code, command, stderr, stdout) -> None:
+        super().__init__(code, command, stderr, stdout)
```

Comment:
I would remove this line because we have a custom `__str__` method and the arguments are already stored on the child object.